nservicebus/messaging/databus.md

```diff
@@ -68,7 +68,22 @@ snippet:MessageWithLargePayloadUsingConvention
 
 NServiceBus DataBus implementations currently behave differently with regard to cleanup of physical attachments used to transfer data properties. `FileShareDataBus` **does not** remove physical attachments once the message is gone. `AzureDataBus` **does** remove Azure storage blobs used for physical attachments once the message is gone.
 
+### Custom cleanup strategy
```

Comment:
Well, you could say the strategy applies to both. When first picking up support cases asking about FileShare not deleting files, I was told "it's not deleting, because you can't know when it should be deleted" and I thought that was logical and came up with some scenarios myself. Then I read that AzureDataBus actually does remove the files. Which seems kind of strange, especially considering the current PR. :)

And also considering this : https://github.com/Particular/NServiceBus/issues/846

I would say both need a cleaning strategy based on x-amount of days, which we can provide, but customers should also be able to provide their own?